src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,35 @@
+package codeu.chat.common;
+
+import codeu.chat.server.Server;
+import codeu.chat.util.Logger;
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+/**
+ * Created by dita on 5/20/17.
+ */
+public final class ServerInfo {
+
+    private final static String SERVER_VERSION = "1.0.0";
+
+    private static final Logger.Log LOG = Logger.newLog(ServerInfo.class);
+
+    public final Uuid version;
+
+    public ServerInfo() {
+        Uuid version;
+
+        try {
+            version = Uuid.parse(SERVER_VERSION);
```

Comment:
Ah, I see. You could also do this with a getter instead, ie get rid of `version` and just add a

```
public Uuid getVersion() {
  try {
    return Uuid.parse(SERVER_VERSION);
  } catch(Exception ex) {
    LOG.error(...);
    return Uuid.NULL;
  }
}
```